docs/readme/cardano-token-deposit-explained.md

```diff
@@ -0,0 +1,36 @@
+
+This way, if you have received 20 NFTs as 20 separate boxes with 1.5 ADA in each box - Yoroi will display that you have ~30 ADA of **locked asset deposit**. So, for example, if you have 100 ADA in total and you see that you have 30 ADA of locked asset deposit - it might be **hard** to spend anything more than 70 ADA. Hard, but still possible, due to repackaging.
+
+## "Repackaging"
+
+The goods can be repackaged from existing boxes to new tighter boxes, though, and the main reason for it would be that storing multiple tokens together in the same box required **SMALLER** minimum deposit, than storing them separately. Storing a single NFT in a box might require 1.5 ADA for each one, but storing 5 NFTs together in one box might require ~3 ADA in total, instead of 7.5 ADA in 5 separate boxes.
+
+This repackaging only can happen in a blockchain transaction, though, so it costs a fee to do it. This is why Yoroi is not doing any repackaging just on its own in the background until you send some transactions. But whenever you **do** send a transaction - a repackaging might happen, which will put your assets tighter together and therefore reduce the locked deposit value, which should not surprise you.
+
+An example of such a reducing case would be if you have 100 ADA and 20 NFTs that you received all separately 1 by 1 - Yoroi will display that you have ~30 ADA of locked asset deposit. But then if you try to send 85 ADA - it might succeed, because then Yoroi will consider doing the repackaging and will estimate that putting all 20 NFTs together will only require ~10 ADA, so spending 85 ADA is fine. After this transaction you will see the deposit value going down from ~30 ADA to ~10 ADA and the total wallet balance is ~15 ADA now. But then next time trying to spend anything more than 4-5 ADA will not be allowed because all the NFTs are already packed together and cannot be repackaged anymore.
+
+It is fairly complicated to make the final estimation of what would be the deposit when the NFTs are already all packed, until you actually try making that transaction, which is why Yoroi would have to display ~30 ADA deposit when the NFTs are all separate in individual boxes initially. And how exactly the repackaging should be done might also depend on user preferences and wallet settings.
+
+But we are working on improving this process now as much as possible in next versions, so this deposit number should now become more and more useful. At the moment the main point is that when you see the **locked asset deposit** value in your wallet - know that it might be **hard or impossible** to reduce it and therefore you might get a **"Not enough funds"** error when trying to send a transaction not leaving enough ADAs in your wallet.
```

Comment:
It might be off-topic for this doc , but one of the follow-up queries for this is typically "if I want to send 10 out of 200 assets to a single destination" or "what can I do to optimise this". Would it be worth sneaking a line about this as a future feature on this page (allowing them more granular control to elect specific multiple assets in single tx to help optimise distribution)?

Reply:
Yes! This is a really good point. We are really in the design process for this feature atm